duplicati.sls

```diff
@@ -1,19 +1,50 @@
+# https://www.duplicati.com
+# https://www.duplicati.com/download
 duplicati:
+  {% for version, type, date, guid32, guid64 in [('2.0.5.1', 'beta', '2020-01-18', '{5BDF9713-1281-4651-AC19-C4D80FE040E7}', '{FC1F2B24-1783-4352-98E9-669896601613}')] %}
+  '{{ version }}':    
+    {% if grains['cpuarch'] == 'AMD64' %}
+    full_name: 'Duplicati 2'
+    installer: 'https://updates.duplicati.com/beta/duplicati{{ version }}_{{ type }}_{{ date }}.x64.msi'
```

Comment:
I think these urls are incorrect. Might want to consider the github links: https://github.com/duplicati/duplicati/releases

Reply:
OK, thanks, I have changed to those.